types/request-promise-native/index.d.ts

```diff
@@ -1,31 +1,75 @@
-// Type definitions for request-promise-native v1.0.3
+// Type definitions for request-promise-native 1.1
```

Comment:
There is no v1.1 of `request-promise-native`: https://github.com/request/request-promise-native/releases